bsoid_app/extract_features.py

```diff
@@ -16,15 +16,18 @@
 from bsoid_app.bsoid_utilities.load_workspace import load_feats, load_embeddings
 from bsoid_app.config import *
 
+framerate = os.environ.get('framerate', '')
+working_dir = os.environ.get('working_dir', '')
+prefix = os.environ.get('prefix', '')
 
 class extract:
```

Comment:
I'd maybe rename these to `Extract` to make them more Python-ic (and also to indicate you touched/modified them.

Reply:
Just flagging this to make sure you didn't miss this and decided to ignore (which is fine!)